lib/ansible/utils/module_docs_fragments/azure.py

```diff
@@ -67,19 +67,27 @@ class ModuleDocFragment(object):
               C(AzureUSGovernment)), or a metadata discovery endpoint URL (required for Azure Stack). Can also be set via credential file profile or
               the C(AZURE_CLOUD_ENVIRONMENT) environment variable.
         default: AzureCloud
+    cli_default_profile:
+        description:
+            - Set to C(true), when login with Azure CLI
```

Comment:
This description doesn't make sense to me.

Reply:
How about `Whether login with Azure CLI`?

Reply:
How about `Whether login with Azure CLI`?

Reply:
`Whether login with Azure CLI` sounds better. 
What happens if this is `false`?
Forgive my basic questions, I've not used Azure

Reply:
`false` is the same as `NONE`, this is a nullable option

Reply:
Has updated